coreneuron/gpu/nrn_acc_manager.cpp

```diff
@@ -41,9 +41,44 @@ void nrn_ion_global_map_delete_from_device();
 void nrn_VecPlay_copyto_device(NrnThread* nt, void** d_vecplay);
 void nrn_VecPlay_delete_from_device(NrnThread* nt);
 
+int cnrn_target_get_num_devices() {
```

Comment:
should it be inside offload.hpp?

Reply:
I think it's OK here. I think we only need to have things in `offload.hpp` if they are used from NMODL.